packages/client/docusaurus/docs/javascript/02-guides/16-closed-captions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+id: closed-captions
+title: Closed captions
+description: How to add closed captions to your calls
+---
+
+The Stream API supports adding real-time closed captions (subtitles for participants) to your calls. This guide shows you how to implement this feature on the client side.
+
+## Call and call type settings
+
+The closed caption feature can be controlled with the following options:
+
+- `available`: the feature is available for your call and can be enabled.
+- `disabled`: the feature is not available for your call. In this case, it's a good idea to "hide" any UI element you have related to closed captions.
+- `auto-on`: the feature is available and will be enabled automatically once the user is connected to the call.
+
+This setting can be set on the call or call type level.
+
+You can check the current value like this:
+
+```typescript
+console.log(call.state.settings?.transcription.closed_caption_mode);
+```
+
+## Closed caption events
+
+If closed captions are enabled for a given call, you'll receive the captions in the `call.closed_caption` WebSocket events. Below, you can find an example payload:
```

Comment:
Let's not mention WebSocket here - from the user standpoint they just do `call.on('call.closed_caption')`, so the WebSocket is an implementation detail.

Reply:
I think they know :D but I understand the argument, fixed in https://github.com/GetStream/stream-video-js/pull/1497/commits/7f2535a66bc182f00a68182bae1ddef0dfe0deb3